Explorer/Assets/DCL/UISystem/UIToolkit/UIToolkit RootCanvas Settings.asset

```diff
@@ -22,7 +22,7 @@ MonoBehaviour:
   m_ReferenceResolution: {x: 1280, y: 720}
   m_ScreenMatchMode: 0
   m_Match: 1
-  m_SortingOrder: 0
+  m_SortingOrder: -200
```

Comment:
Did you check if the ui is now behind the name tags? Is that expected?

Reply:
But the name tags are in the 3D world, aren't they?

Reply:
Yep, you should be right...